mindjunkies/courses/migrations/0004_coursetoken_intro_video_alter_coursetoken_motivation.py

```
@@ -0,0 +1,24 @@
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='coursetoken',
+            name='intro_video',
+            field=cloudinary.models.CloudinaryField(default='', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='coursetoken',
+            name='motivation',
+            field=models.TextField(help_text='Write your motivation or plan for the course, including how you plan to organize it.'),
```

Ruff (0.8.2)

Line too long (133 > 119)

(E501)